.github/workflows/stale.yml

```diff
@@ -0,0 +1,68 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: "0 12 * * 0"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      # 120+30 day stale policy for PRs
+      # * Except PRs marked as "no stale"
+      - name: Stale PRs policy
+        uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          exempt-pr-labels: "no stale"
+          days-before-stale: 120
+          days-before-close: 30
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          remove-stale-when-updated: true
+          stale-pr-label: "stale"
+          stale-pr-message: >
+            There hasn't been any activity on this pull request recently. It has been
+            automatically marked as stale because of that and it will be closed if no
+            further activity occurs in the next 30 days.
+
+      # 180+30 day stale policy for open issues
+      # * Except Issues marked as "no stale"
+      - name: Stale Issues policy
+        uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          exempt-issue-labels: "no stale,needs more information"
+          days-before-stale: 180
+          days-before-close: 30
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          remove-stale-when-updated: true
+          stale-issue-label: "stale"
+          stale-issue-message: >
+            There hasn't been any activity on this issue in the past 6 months, so it has
+            been automatically marked as stale and it will be closed automatically if no
+            further activity occurs in the next 30 days.
+
+            If you think this is a mistake and want to keep the issue open, please ask a
+            PSC member to mark it as "no stale".
```

Comment:
Why is PSC member intervention mentioned for issues and not PRs ?

Reply:
Added the same message to PRs :+1: and squashed everything

Reply:
Ok, thanks. Just to be sure: a simple comment is going to unstale it, correct ?

Reply:
That's right, according to their docs:

> If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart

Reply:
Ok thanks. So since it was not entirely obvious to me, we could may be replace 

*If you think this is a mistake and want to keep the issue open, please ask a PSC member to mark it as "no stale.* 

by 

*If you want this issue to never become stale, please ask a PSC member to apply the "no stale" label.* ?

Reply:
done :+1: